content/docs/learning-guide/tutorials/environments/create-terrain-from-images/texture-terrain.md

```diff
@@ -0,0 +1,235 @@
+
+1. Create a child entity of the Terrain Spawner entity named `Terrain Rock` and add an [**Invert Gradient Modifier**](/docs/user-guide/components/reference/gradient-modifiers/invert-gradient-modifier) component and a **Shape Reference** component to it.
+
+2. On the Shape Reference component, click the {{< icon "picker.svg" >}} **Picker** button, then click the Terrain Spawner entity in Entity Outliner to create a reference.
+
+3. On the Invert Gradient Modifier component, click the {{< icon "picker.svg" >}} **Picker** button, then click the Terrain Grass entity in Entity Outliner to create a reference.
+
+#### Add a Terrain Surface Gradient List
+
+These steps associate the gradient entities with the surface tag names you created.
+
+1. With the Terrain Spawner selected, add a **Terrain Surface Gradient List** component.
+
+2. In the **Terrain Surface Gradient List** component, click the {{< icon "add.svg" >}} **Add** button to add a new gradient slot.
+
+3. To the right of the **Gradient Entity** property, click the {{< icon "picker.svg" >}} **Picker** button, then click the Terrain Grass entity in Entity Outliner to reference it.
+
+4. For the **Surface Tag** property, select the `tutorial_grass` surface tag name from the list. Note that the list contains the surface tag names you created in Asset Editor.
+
+5. Repeat steps 2 though 4 to add the Terrain Rock entity and associate it with the `tutorial_rock` surface tag.
+
+#### Add surface weights to the Terrain Surface Materials List
+
+This assigns the detail materials to the terrain based on the surface tag names and associated gradient entities.
+
+1. Select the Terrain Spawner entity in the Entity Outliner.
+
+2. In the **Terrain Surface Materials List** component, clear the entry for the default material. Our surface maps will cover the entire terrain, so the default material will no longer be necessary.
+
+3. To the right of the **Material Mappings** property group, click the {{< icon "add.svg" >}} **Add** button to add a new material slot.
+
+4. In the **Surface Tag** property, select `tutorial_grass`.
+
+5. To the right of the **Material asset** property, click the {{< icon "file-folder.svg" >}} **File** button and select the `forest_ground_01` material asset to associate with the surface tag.
+
+6. Repeat steps 3 through 5 to add `tutorial_rock` and associate it with the `rock_boulder_dry` material asset.
+
+As you add detail materials and surface tag name pairs to the **Terrain Surface Materials List** component, the materials appear on the terrain surface.
+
+{{% /tab %}}
+
+{{% tab name="Terrain texturing with Landscape Canvas" %}}
+
+In this section, you'll color and surface the terrain using Landscape Canvas.
+
+1. If you have already followed through the "Terrain texturing with entities" tab, you will need to reload your level without saving to revert the changes that you made.
```

Comment:
Nice tip starting off here! Maybe we should remind the user at the beginning of the "Terrain texturing with entities" to save before they begin so they can easily revert?

Reply:
Added callout at the beginning of "Terrain texturing with entities" and then again after tuning the rendering settings.